test/extensions/quic_listeners/quiche/active_quic_listener_config_test.cc

```diff
@@ -29,6 +33,9 @@ TEST(ActiveQuicListenerConfigTest, CreateActiveQuicListenerFactory) {
     idle_timeout: {
       seconds: 2
     }
+    enabled:
+      default_value: true
+      runtime_key: foo_key
```

Comment:
What would happen if you leave this out? What's the default key and value?

Reply:
@danzh2010 on this test level is not possible to verify value of feature flag, as we are only testing how yaml is parsed in corresponding code/proto representaion, and proto for RuntimeFeatureFlag looks like this:
```
message RuntimeFeatureFlag {
  google.protobuf.BoolValue default_value = 1 [(validate.rules).message = {required: true}];
  string runtime_key = 2 [(validate.rules).string = {min_bytes: 1}];
}
```
Actual value of feature flag is obtained from `Runtime` primitive (which is available in active_quic_listener test). Absence of feature flag configuration is covered by test: 
https://github.com/envoyproxy/envoy/pull/9679/files#diff-6a27a42427472c010c1a55f272fa72f4R272